spec/database_cleaner/configuration_spec.rb

```diff
@@ -174,15 +178,22 @@ def cleaning &block
       context "multiple orm proxy methods" do
         class FakeStrategy < Struct.new(:orm, :db, :strategy); end
 
-        context "with differing orms and dbs" do
-          let(:active_record_1) { FakeStrategy.new(:active_record) }
-          let(:active_record_2) { FakeStrategy.new(:active_record, :different) }
-          let(:data_mapper_1)   { FakeStrategy.new(:data_mapper) }
+        context "with differing orms and dbs but same strategy" do
+          let(:active_record_1) { FakeStrategy.new(:active_record, nil, :truncation) }
+          let(:active_record_2) { FakeStrategy.new(:active_record, :different, :truncation) }
+          let(:data_mapper_1)   { FakeStrategy.new(:data_mapper, nil, :truncation) }
 
           before do
             config.stub_cleaners [active_record_1,active_record_2,data_mapper_1]
           end
 
+          it "should proxy #strategy to all cleaners and return a single result" do
+            expect(config.strategy)
+              .to eq(active_record_1.strategy)
+              .and eq(active_record_2.strategy)
+              .and eq(data_mapper_1.strategy)
+          end
```

Comment:
I'm a little confused about this assertion. It seems to imply that:
```ruby
config.strategy == active_record_1.strategy &&
  config.strategy == active_record_2.strategy && 
  config.strategy == data_mapper_1.strategy
```
which also seems to imply that all three fake strategies are equal to each other? That doesn't seem quite right. But I might be misunderstanding something here.

Reply:
Ohh, I think I get it. It's saying that the strategy is `:truncation` for all of them, and even though the ORMs differ, the strategy is the same. Hence the `uniq` call in the code. Got it.